exercise/models.py

```diff
@@ -164,7 +164,7 @@ def reveal_time(self):
     @property
     def exercise_specific_properties(self):
         email_properties = ExerciseEmailProperties.objects.filter(
-            email_id=self.id, exercise__emails__id=self.id
+            email_id=self.id, exercise=self.exercise_set.first()
```

Comment:
We need to narrow down to 1 specific exercise because there only ever going to be 1 prop records per exercise-email pair. There's no way to create more than 1 properties for the same email in any given exercise as of now in the admin panel, but if you wanted to enforce uniqueness we could update the model with unique together.
```python
def exercise_specific_properties(self, exercise=None):
    email_properties = ExerciseEmailProperties.objects.filter(
        email=self, exercise=exercise
    ).first()
```